src/java/fr/paris/lutece/portal/web/dashboard/AdminDashboardJspBean.java

```diff
@@ -94,7 +94,7 @@
     // Jsp
     private static final String JSP_MANAGE_DASHBOARDS = "ManageAdminDashboards.jsp";
     private static final String EMPTY_STRING = "";
-    private AdminDashboardService _service = AdminDashboardService.getInstance( );
+    private transient AdminDashboardService _service = AdminDashboardService.getInstance( );
```

Comment:
As before, this does not make the class properly serializable.

Reply:
Will be corrected